backend/src/test/java/de/amos/apachepulsarui/service/TenantServiceIntegrationTest.java

```diff
@@ -47,14 +50,35 @@ void getAllTenants_returnsTenants() {
                 .findFirst().orElseThrow();
         Assertions.assertThat(tenantX).isNotNull();
         Assertions.assertThat(tenantY).isNotNull();
-        TenantDto tenantDetailsX = tenantService.getTenantDetails(tenantX);
-        TenantDto tenantDetailsY = tenantService.getTenantDetails(tenantY);
+        TenantDetailsDto tenantDetailsX = tenantService.getTenantDetails(tenantX);
+        TenantDetailsDto tenantDetailsY = tenantService.getTenantDetails(tenantY);
         Assertions.assertThat(tenantDetailsX.getNamespaces())
                 .containsExactlyInAnyOrder("tenantX/namespace1", "tenantX/namespace2");
         Assertions.assertThat(tenantDetailsY.getNamespaces())
                 .containsExactlyInAnyOrder("tenantY/namespace1");
     }
 
+    @Test
+    void getAllTenatsFiltered() {
+        List<TenantDto> tenants = tenantService.getAllFiltered(Collections.emptyList());
+        var tenantY = tenants.stream()
```

Comment:
You could also create TenantDtos and then assert that the result from `getAllFiltered` contains exactly these dtos : ) I think it would be easier to understand as a test.